app/styles/components/c-cover_page.scss

```diff
@@ -1,7 +1,7 @@
 @import '../base/text.scss';
 @import '../base/colors.scss';
 
-.c_cover_page {
+.c-cover-page {
```

Comment:
why did you change this? all other files in the project use underscores, we should keep consisntency